Workaround --linux_perf_bts_block flag breakage.

Mateusz reports that running honggfuzz fails as:
    [2016-11-11T21:54:27+0000][W][1190] arch_perfOpen():223 mmap(mmapAuxBuf) failed,
    try increasing the kernel.perf_event_mlock_kb sysctl (up to even 300000000): Cannot allocate memory

It seems aux data also needs WRITE permissions.

Reported-by: Mateusz Lenik
diff --git a/linux/perf.c b/linux/perf.c
index d8ede5f..2f71b3d 100644
--- a/linux/perf.c
+++ b/linux/perf.c
@@ -217,3 +217,4 @@ static bool arch_perfOpen(honggfuzz_t * hfuzz, fuzzer_t * fuzzer UNUSED, pid_t p
     fuzzer->linux.perfMmapAux =
-        mmap(NULL, pem->aux_size, PROT_READ, MAP_SHARED, *perfFd, pem->aux_offset);
+        mmap(NULL, pem->aux_size, PROT_READ | PROT_WRITE, MAP_SHARED, *perfFd, pem->aux_offset);
+
     if (fuzzer->linux.perfMmapAux == MAP_FAILED) {
